src/concrete/ml/quantization/quantized_module.py

```diff
@@ -730,8 +824,15 @@ def compile(
         # Quantize the inputs
         q_inputs = self.quantize_input(*inputs)
 
+        # Make sure all inputs are quantized to int64 and are not None
+        assert all_values_are_of_dtype(
+            *to_tuple(q_inputs), dtypes="int64", allow_none=False
+        ), "Inputs were not quantized to int64"
+
         # Generate the input-set with proper dimensions
-        inputset = _get_inputset_generator(q_inputs)
+        # Ignore [arg-type] check from mypy as it is not able to see that no values in `q_inputs`
+        # is None
+        inputset = _get_inputset_generator(q_inputs)  # type: ignore[arg-type]
 
```

Comment:
Here also we should probably just do

```python
assert all(item is not None for item in q_inputs)
```

Reply:
mypy still complains after that unfortunately

Reply:
that is weird. Then yeah not choice but to ignore